spec/models/instance_spec.rb

```diff
@@ -3,7 +3,10 @@
 RSpec.describe Instance, type: :model do
   it { is_expected.to have_many(:instance_users) }
   it { is_expected.to have_many(:users).through(:instance_users) }
-  it { is_expected.to have_many(:announcements).class_name(Instance::Announcement.name) }
+  it do
+    is_expected.to have_many(:announcements).class_name(Instance::Announcement.name).
```

Comment:
Yah, I'm also not very comfortable about it.

But this is the syntax(or problem?) of shoulda-matchers gem, see the conditions section, they also use <code> it do ... end </code>
http://matchers.shoulda.io/docs/v2.8.0/Shoulda/Matchers/ActiveRecord.html#belong_to-instance_method 

When using <code>expect</code> we need a <code>subject</code> which is not defined.